### ISSUE TITLE
Error in ng serve - Posible solution: Upgrade AutoPrefixer

### ISSUE THREAD

**Issue by MrJmpl3** [angular/angular-cli#9861]
### Versions
```
/ \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
/ △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
/ ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
/_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
|___/

Angular CLI: 1.7.2
Node: 8.9.4
OS: win32 x64
Angular: 5.2.7
... animations, common, compiler, compiler-cli, core, forms
... http, language-service, platform-browser
... platform-browser-dynamic, platform-server, router

@angular/cli: 1.7.2
@angular-devkit/build-optimizer: 0.3.2
@angular-devkit/core: 0.3.2
@angular-devkit/schematics: 0.3.2
@ngtools/json-schema: 1.2.0
@ngtools/webpack: 1.10.1
@schematics/angular: 0.3.2
@schematics/package-update: 0.3.2
typescript: 2.5.3
webpack: 3.11.0

```

### Repro steps

* ng serve

### Observed behavior
```
ERROR in ./node_modules/raw-loader!./node_modules/postcss-loader/lib??embedded!./node_modules/sass-loader/lib/loader.js??ref--8-3!./src/styles.scss
Module build failed: TypeError: Cannot read property 'warn' of undefined
    at Declaration.warn (D:\_JOSE\PROYECTOS-JAVASCRIPT\PROJECT\node_modules\postcss\lib\node.js:153:24)
    at GridTemplateAreas.insert (D:\_JOSE\PROYECTOS-JAVASCRIPT\PROJECT\node_modules\@angular\cli\node_modules\autoprefixer\lib\hacks\grid-template-areas.js:121:18)
    at GridTemplateAreas.add (D:\_JOSE\PROYECTOS-JAVASCRIPT\PROJECT\node_modules\@angular\cli\node_modules\autoprefixer\lib\declaration.js:225:21)
    at GridTemplateAreas.process (D:\_JOSE\PROYECTOS-JAVASCRIPT\PROJECT\node_modules\@angular\cli\node_modules\autoprefixer\lib\prefixer.js:168:22)
    at GridTemplateAreas.process (D:\_JOSE\PROYECTOS-JAVASCRIPT\PROJECT\node_modules\@angular\cli\node_modules\autoprefixer\lib\declaration.js:235:41)
    at Supports.prefixed (D:\_JOSE\PROYECTOS-JAVASCRIPT\PROJECT\node_modules\@angular\cli\node_modules\autoprefixer\lib\supports.js:94:50)
    at D:\_JOSE\PROYECTOS-JAVASCRIPT\PROJECT\node_modules\@angular\cli\node_modules\autoprefixer\lib\supports.js:319:39
    at Array.map (<anonymous>)
    at Supports.add (D:\_JOSE\PROYECTOS-JAVASCRIPT\PROJECT\node_modules\@angular\cli\node_modules\autoprefixer\lib\supports.js:317:22)
    at Supports.process (D:\_JOSE\PROYECTOS-JAVASCRIPT\PROJECT\node_modules\@angular\cli\node_modules\autoprefixer\lib\supports.js:344:20)
    at D:\_JOSE\PROYECTOS-JAVASCRIPT\PROJECT\node_modules\@angular\cli\node_modules\autoprefixer\lib\processor.js:44:37
    at D:\_JOSE\PROYECTOS-JAVASCRIPT\PROJECT\node_modules\postcss\lib\container.js:292:28
    at D:\_JOSE\PROYECTOS-JAVASCRIPT\PROJECT\node_modules\postcss\lib\container.js:144:26
    at Root.each (D:\_JOSE\PROYECTOS-JAVASCRIPT\PROJECT\node_modules\postcss\lib\container.js:110:22)
    at Root.walk (D:\_JOSE\PROYECTOS-JAVASCRIPT\PROJECT\node_modules\postcss\lib\container.js:143:21)
    at Root.walkAtRules (D:\_JOSE\PROYECTOS-JAVASCRIPT\PROJECT\node_modules\postcss\lib\container.js:290:25)

```

### Desired behavior

Upgrade AutoPrefixer

**Comment by tbimene** [comment]
Hi I just installed angular and I get this version and I don't understand what version is this supposed to be?

**Comment by rahulbhooteshwar** [comment]
@MrJmpl3 It is not specific to Angular CLI.  From the error it looks like it is something to do with your code ? You should check if you are trying to access **'warn'** property on some null value.

**Comment by elclanrs** [comment]
CLI should update autoprefixer to 8.1. I got this same issue, and It is a bug that comes from autoprefixer. I opened an issue in their repo here https://github.com/postcss/autoprefixer/issues/988 and it is fixed in 8.1 now.
